mitodl/open-discussions
# Added podcasts & podcast episodes to index

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2747 

#### What's this PR do?
Adds podcasts and podcast episodes to the ES index

#### How should this be manually tested?
Add some `Podcast`s and `PodcastEpisode`s, then run the `recreate_index` management command. You should see those objects in the index. This query will show just the podcast and episodes: `{ "query" : { "match" : { "object_type": "podcast" } } }`

#### Any background context you want to provide?
I made some changes to the Dockerfile and docker-compose yml for running the app in a notebook to get it working. If it was working for other folks on the project and I have some machine-specific issue, I can revert those changes
